packages/app/src/app/pages/Dashboard/Content/routes/RepositoryBranches/index.tsx

```diff
@@ -48,12 +51,13 @@ export const RepositoryBranchesPage = () => {
       return [{ type: 'skeleton-row' }, { type: 'skeleton-row' }];
     }
 
-    const branchItems: DashboardGridItem[] = repositoryProject.branches.map(
-      branch => ({
-        type: 'branch',
-        branch,
-      })
-    );
+    const branches = [...repositoryProject.branches];
+    const orderedBranches = branches.sort(sortByLastAccessed);
```

Comment:
Any particular reason why `branches` is created?

Reply:
sort mutates the array, and branches is the direct reference of the data coming from overmind/graphql.